EIPS/eip-4362.md

```diff
@@ -0,0 +1,279 @@
+---
+eip: 4362
+title: SIWE CapGrok Extension
```

Comment:
```suggestion
title: EIP-4361 Capabilities Extension
```
Slightly more descriptive title.

Reply:
Thanks for the recommendation! I changed the title to EIP-4361 Capabilities Extension (CapGrok).

Reply:
Does this sound ok to you @Pandapip1 ?